api/conf/schema.json

```diff
@@ -2642,7 +2723,7 @@
 								"type": "integer"
 							},
 							"vars": {
-								"items": {
+								"item": {
```

Comment:
I tried edit `fault-injection` plugin in FE, an error occured:

![image](https://user-images.githubusercontent.com/2561857/119213141-9f730f00-baef-11eb-86e5-4ec7f30ad937.png)

refer to 
* https://json-schema.org/understanding-json-schema/reference/array.html

the reason is that in JSON Schema, it should use `items` to define the array item type

and after I changed `item` to `items`, Fe passed.

So It should rollback this change IMO

Reply:
thanks.